Main.java

```diff
@@ -9,7 +10,39 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // Determines if number is a Fizz, Buzz, or FizzBuzz
+
+    // Create a scanner for user input
+    Scanner input = new Scanner(System.in);
+    
+    // prompt the user for a number
+    System.out.print("Please enter a number:");
+
+    // get the number from the user
+    int number = input.nextInt();
+
+    // determine weather the number is a Fizz
+    int fizz = 0;
+
+    int remainderFizz = number%3;
+
+    int remainderBuzz = number%5;
+
+    // determine if the number is Fizz, Buzz, or both    
+    if(remainderFizz==fizz){
+      // tell them if the number is Fizz
+      System.out.print("This number is Fizz");
+    }
+    if(remainderBuzz==fizz){
+      // tell them if the number is Buzz
+      System.out.print("This number is Buzz");
+    }
+    if(remainderBuzz == fizz && remainderFizz == fizz){
+      // tell them if the number is FizzBuzz
+      System.out.print("This number is FizzBuzz");
+    }
```

Comment:
Because these are not if-else if commands and they are not ordered correctly, if you type in 15, it would say fizz, buzz, fizzbuzz. It should only say 1 of them. Also, you are missing the case that it would just print out the number is it is not a fizz or a buzz.